manifests/init.pp

```diff
@@ -74,7 +74,8 @@
   String[1] $root_store            = 'LocalMachine',
   String[1] $store_dir             = 'My',
   Stdlib::Windowspath $scripts_dir = 'C:\temp',
-  Boolean $exportable              = true
+  Boolean $exportable              = true,
+  String $grant_user               = undef,
```

Comment:
When the default value is undef, the variable needs to be optional. Also IF a string is provided, you can enforce the minimal string length:
```suggestion
  Optional[String[1]] $grant_user  = undef,
```